src/main/java/io/github/fablabsmc/fablabs/api/provider/v1/ContextKey.java

```diff
@@ -13,8 +13,7 @@
  */
 public final class ContextKey<C> {
     public static final ContextKey<@Nullable Void> NO_CONTEXT = create(Void.class, new Identifier("fabric:no_context"));
-    private static final Map<Class<?>, Map<Identifier, ContextKey<?>>> contextKeys = new HashMap<>();
-
+    private static final Map<Class<?>, Map<Identifier, ContextKey<?>>> CONTEXT_KEYS = new HashMap<>();
```

Comment:
It's not really a constant, though... I'm not sure, but wouldn't the naming conventions suggest leaving this in lower camel case?

Reply:
static final fields in yarn standards are typically in upper camel case